README.md

```diff
@@ -0,0 +1,189 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Linters</summary>
+  <ul>
+    <li>Rubocop</li>
+    <li>Style lint</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Use of Ruby on rails]**
+- **[Use of device to user autenticate]**
+- **[Use of Rspec and capybara for testing]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO 
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://reactcapstonproject.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+You need ruby and bundler
+
+### Setup
+
+Clone this repository to your desired folder:
+
+sh <br>
+cd my-folder <br>
+git clone https://github.com/ClaudiaRojasSoto/Recipe_App.git
+
+### Install
+
+Install this project with:
+
+- gem install rails
+- bundle install
+
+### Usage
+
+To run the project, execute the following command:
+
+rails server
+
+### Run tests
+
+To run tests, run the following command:
+
+rspec spec
+
+<!-- ### Deployment
+
+You can deploy this project using: -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Andy Zam**
+
+- GitHub: [Andres Zamorano](https://github.com/Dachrono)
+- Twitter: [Andres Zamorano](https://twitter.com/Dachrono)
+- LinkedIn: [Andres Zamorano](https://www.linkedin.com/in/andres-zamorano-785b77a1/)
+
+👤 **Claudia Rojas**
+
+- GitHub: @ClaudiaRojas
+- LinkedIn: @ClaudiaRojas
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [1] **[Modify you recipe foods.]**
+- [2] **[Organize your recipes for type.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ClaudiaRojasSoto/Blog_App/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please feel free to send me corrections for make it better I would feel glad to read your comments.
+And think If you enjoy gift me a star.  
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
```

Comment:
- [ ] I suggest that you acknowledge anyone that  inspire instead of leaving your acknowledge section blank :blush:

Reply:
Done!